content/applications/inventory_and_mrp/manufacturing/workflows/continuous_improvement.rst

```diff
@@ -0,0 +1,233 @@
+   customers describe it as "sleek and elegant". However, recent customer feedback about the coat
+   rack has brought to light quality issues that necessitate revising the current manufacturing
+   process.
+
+   The product revision workflow begins when the customer service team receives a ticket in the
+   Helpdesk app from a customer having problems with the coat rack she purchased. The customer,
+   Abigail Peterson, has found that her coat rack falls over when more than five coats are hanging
+   from it. This is a major issue, as the coat rack has enough dowels for six coats.
+
+   .. image:: continuous_improvement/helpdesk-ticket.png
+      :align: center
+      :alt: A Helpdesk ticket about an issue with the coat rack product.
+
+   Marc, the customer service employee assigned to the helpdesk ticket, opens the *Quality* app and
+   creates a new quality alert. He tags the *Production Quality Team* and assigns Julie Andreson as
+   the quality employee responsible for the alert.
+
+   Julie reviews the alert, and consults with her team about the best course of action. They decide
+   that it is necessary to revise the product's |BoM| to prevent the issue from occurring in the
+   future, which Julie notes in the :guilabel:`Corrective Actions` tab of the quality alert.
+
+   .. image:: continuous_improvement/quality-alert.png
+      :align: center
+      :alt: A quality alert created about the issue with the coat rack product.
+
+   Julie then messages product engineer Joe Kazan in the chatter of the quality alert to bring it to
+   his attention. Joe opens the |PLM| app and creates a new |ECO|, noting the problem with the coat
+   rack and suggesting that a change to the product's |BoM| may be necessary.
+
+   .. image:: continuous_improvement/eco.png
+      :align: center
+      :alt: An ECO created to update the coat rack product's BoM.
+
+   Joe clicks *Start Revision*, and then the *Revision* smart button to open version two of the coat
+   rack's |BoM|. This |BoM| was created alongside the |ECO|, and remains archived until it is
+   approved.
+
+   After some testing, Joe discovers that adding a metal *support rod* to the coat rack strengthens
+   it, allowing the rack to hold six or more coats without falling over. He updates the |BoM| to
+   include the support rod as one of the components, and adds an extra operation to make sure it is
+   installed during the manufacturing process. Finally, he leaves a message in the chatter of the
+   |ECO|, letting his manager Jose know that it is ready for review.
+
+   .. image:: continuous_improvement/bom.png
+      :align: center
+      :alt: The coat rack BoM, updated to add an extra component and operation.
+
+   Jose reviews the changes and confirms they are an effective method for addressing the problem
+   with the coat rack. He moves the |ECO| to the *Approved* stage, which makes version two of the
```

Comment:
again, it might be beneficial to add a screenshot of this -- showing the 'Approved' stage -- it would provide more visual context to the flow, which would be super helpful to the reader.